packages/block-library/src/form/edit.js

```diff
@@ -130,8 +130,7 @@ const Edit = ( { attributes, setAttributes, clientId } ) => {
 			{ submissionMethod !== 'email' && (
 				<InspectorControls group="advanced">
 					<SelectControl
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
Enable "Form and input blocks" in Gutenberg Experimental settings. Insert Form block. Choose a custom submission method.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/3c37de18-6922-44fe-913a-652c70ea6084" alt="Form block inspector, before" width="271">|<img src="https://github.com/user-attachments/assets/5c0aee8e-4a47-40d0-b609-2e25f65a7df5" alt="Form block inspector, after" width="271">|